www/__tests__/customMetricsHelper.test.ts

```diff
@@ -0,0 +1,51 @@
+import { getConfig } from '../js/config/dynamicConfig';
+import {
+  getCustomFootprint,
+  getCustomMETs,
+  initCustomDatasetHelper,
+} from '../js/metrics/customMetricsHelper';
+import { mockBEMUserCache } from '../__mocks__/cordovaMocks';
+import { mockLogger } from '../__mocks__/globalMocks';
+import fakeLabels from '../__mocks__/fakeLabels.json';
+
+mockBEMUserCache();
+mockLogger();
+
+global.fetch = (url: string) =>
+  new Promise((rs, rj) => {
+    setTimeout(() =>
+      rs({
+        text: () =>
+          new Promise((rs, rj) => {
+            let myJSON = JSON.stringify(fakeLabels);
+            setTimeout(() => rs(myJSON), 100);
+          }),
+      }),
+    );
+  }) as any;
+
+it('gets the custom mets', async () => {
+  const appConfig = await getConfig();
+  await initCustomDatasetHelper(appConfig);
+  expect(getCustomMETs()).toMatchObject({
+    walk: expect.any(Object),
+    bike: expect.any(Object),
+    bikeshare: expect.any(Object),
```

Comment:
I probably did not do this originally, because I was mostly trying to test the keys. The default keys are `WALKING`, `BICYCLING`, etc, while the custom keys are `walk`, `bike`, etc. 

I updated the footprint test, since that is a single value, but I have not updated the MET testing, since those objects are complex (some are speed dependent, etc), and taking the time to copy them over from `fakeLabels.json` seems like it would do more to test the test than the code, but I'll add a note about checking the keys!